docs/root/version_history/v1.13.0.rst

```diff
@@ -4,82 +4,82 @@
 Changes
 -------
 
-* access log: added FILTER_STATE :ref:`access log formatters <config_access_log_format>` and gRPC access logger.
-* admin: added the ability to filter :ref:`/config_dump <operations_admin_interface_config_dump>`.
-* access log: added a :ref:`typed JSON logging mode <config_access_log_format_dictionaries>` to output access logs in JSON format with non-string values
-* access log: fixed UPSTREAM_LOCAL_ADDRESS :ref:`access log formatters <config_access_log_format>` to work for http requests
+* access log: added FILTER_STATE :ref:`access log formatters <v1.13.0:config_access_log_format>` and gRPC access logger.
+* admin: added the ability to filter :ref:`/config_dump <v1.13.0:operations_admin_interface_config_dump>`.
+* access log: added a :ref:`typed JSON logging mode <v1.13.0:config_access_log_format_dictionaries>` to output access logs in JSON format with non-string values
+* access log: fixed UPSTREAM_LOCAL_ADDRESS :ref:`access log formatters <v1.13.0:config_access_log_format>` to work for http requests
 * access log: added HOSTNAME.
 * api: remove all support for v1
-* api: added ability to specify `mode` for :ref:`Pipe <envoy_api_field_core.Pipe.mode>`.
+* api: added ability to specify `mode` for :ref:`Pipe <v1.13.0:envoy_api_field_core.Pipe.mode>`.
 * api: support for the v3 xDS API added. See :ref:`api_supported_versions`.
-* aws_request_signing: added new alpha :ref:`HTTP AWS request signing filter <config_http_filters_aws_request_signing>`.
```

Comment:
afaict this was added erroneously to the v1.13 changelog when v1.14 was current - consequently the link doesnt work in v1.13

ive removed the link - the other alternative is to move this to the v1.14 changelog